static/js/src/interfaces/components/InterfaceDetails/InterfaceDetails.tsx

```diff
@@ -217,6 +217,14 @@ function InterfaceDetails() {
                   &nbsp;&nbsp;Submit a bug
                 </a>
               </p>
+              {!isCommunity && (
+              <p>
+                <a href={`https://github.com/canonical/charm-relation-interfaces/tree/main/interfaces/${interfaceName}${interfaceData?.version ? `/${interfaceData.version}` : ""}`}>
```

Comment:
aaactually, this should not include the `/${interfaceData.version}` at the end of the url, it should go to the folder that contains the version folders

Reply:
I removed it too :)